revista-latinoamericana-de-metalurgia-y-materiales.csl

```diff
@@ -0,0 +1,312 @@
+                <text term="presented at"/>
+                <text variable="event"/>
+              </group>
+              <text variable="event-place"/>
+            </group>
+          </else>
+        </choose>
+      </if>
+    </choose>
+  </macro>
+  <macro name="access">
+    <choose>
+      <if type="webpage">
+        <choose>
+          <if variable="URL">
+            <group delimiter=". ">
+              <text term="online" prefix="[" suffix="]"/>
+              <group delimiter=": ">
+                <text term="available at" text-case="capitalize-first"/>
+                <text variable="URL"/>
+              </group>
+              <group prefix="[" suffix="]" delimiter=": ">
+                <text term="cited"/>
+                <date variable="accessed">
+                  <date-part name="day" form="numeric-leading-zeros" suffix="-"/>
+                  <date-part name="month" form="short" suffix="-" strip-periods="true"/>
+                  <date-part name="year" form="long"/>
+                </date>
+              </group>
+            </group>
+          </if>
+        </choose>
+      </if>
+    </choose>
+  </macro>
+  <macro name="page">
+    <choose>
+      <if type="article-journal" match="any">
+        <text variable="page"/>
+      </if>
+      <else-if type="book" match="any">
+        <label plural="always" suffix=" " variable="page" form="short"/>
+        <text variable="number-of-pages"/>
+      </else-if>
+      <else-if type="chapter" match="any">
+        <label plural="never" suffix=" " variable="page" form="short"/>
+        <text variable="page"/>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="locators">
+    <choose>
+      <if type="article-journal">
+        <text variable="volume" suffix=" "/>
+        <text variable="issue" prefix="(" suffix="): "/>
+      </if>
+      <else-if type="chapter paper-conference" match="any">
+        <text macro="publisher"/>
+      </else-if>
+      <else>
+        <group delimiter=", ">
+          <text macro="publisher"/>
+          <text macro="issued"/>
+          <text macro="page"/>
+        </group>
+        <group>
+          <text macro="edition"/>
+          <group suffix=": ">
+            <number suffix=" " variable="volume"/>
+            <number prefix="(" suffix=")" variable="issue"/>
+          </group>
+        </group>
+      </else>
+    </choose>
+  </macro>
+  <citation collapse="citation-number">
+    <sort>
+      <key variable="citation-number"/>
+    </sort>
+    <layout prefix="[" suffix="]" delimiter=",">
+      <text variable="citation-number"/>
+    </layout>
+  </citation>
+  <bibliography entry-spacing="0" second-field-align="flush">
+    <layout suffix=".">
+      <text variable="citation-number" prefix="[" suffix="]."/>
+      <text macro="author" suffix=", "/>
+      <choose>
+        <if type="article-journal">
+          <group>
+            <text macro="title" font-style="normal" suffix=", "/>
```

Comment:
in the articles I looked at & styleguide, there is not article title at all for journal articles. Remove?